e2e/features/OwnerCRUD.feature

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright 2024 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# author Holger Mayer
+#
+@CRUD
+Feature: CRUD Owner
+
+Background: Preconditions for testing
+    Given I reset the test data
+    And   I launch the pet clinic application   
+    
+Scenario: Add new owner
+    When I want to create a new pet owner in the application
+    And I enter the pet owner with first name "Travis" last name "Peterson" address "1234 Fake St" city "Springfield" telephone "1234567890"
+    And I save the new pet owner record
+    Then I see the owner "Travis Peterson" in the pet owner list
+    And I see the owner details with first name "Travis" last name "Peterson" address "1234 Fake St" city "Springfield" telephone "1234567890"
+
+
+Scenario: Edit  owner
+    When I want to create a new pet owner in the application
+    And I enter the pet owner with first name "Clark" last name "Kent" address "1234 Fake St" city "Springfield" telephone "1234567890"
+    And I save the new pet owner record
+    And I select the owner "Clark Kent"
+    And I begin editing the owner record
+    And I enter the pet owner with first name "Travis" last name "Peterson" address "North Park Blvd" city "Bielefeld" telephone "49492020"
+    And I save the changed pet owner record
+    Then I see the owner "Travis Peterson" in the pet owner list
+    And I see the owner details with first name "Travis" last name "Peterson" address "North Park Blvd" city "Bielefeld" telephone "49492020"
+
+###Information:
+### Scenario: Delete owner
```

Comment:
We could create an issue to add this feature.

Reply:
This feature could be added to the issue list and implemented later. I recommend that we first get the UI e2e tests up an running. I also think it would be better to add API tests before changing or adding functionality.

Reply:
Yes I am agree. Creating the issue will allow us to have this evolution in our backlog